motionplan/motionPlanner.go

```diff
@@ -517,12 +518,12 @@ func CheckPlan(
 	if sfPlanner.planOpts, err = sfPlanner.plannerSetupFromMoveRequest(
 		currentPosition,                    // starting pose
 		planNodes[len(planNodes)-1].Pose(), // goalPose
-		plan[0],                            // starting configuration
+		sf.sliceToMap(currentInputs),       // starting configuration
```

Comment:
What is this change for?

Reply:
Is this fixing a bug? If so, we should create a PR for it separate from this one, in case this one takes longer in review.

Reply:
yep, this line is responsible for fixing a bug.
I will go ahead and create a PR for this along with a JIRA ticket.

Reply:
thank you!

Reply:
Tracking issue in this PR: https://github.com/viamrobotics/rdk/pull/3123

Reply:
Reverted